update-since-todo.py

```diff
@@ -53,12 +53,26 @@ def analyze_file(file, lineno, commits_and_tags, dry_run=False):
         print(f"\tfirst tag was {first_tag}")
         commits_and_tags[line_sha] = first_tag
         if not dry_run:
+            since_version = first_tag.replace("jenkins-", "")
             update_file(
                 file,
                 int(lineno),
                 "@since TODO",
-                "@since " + first_tag.replace("jenkins-", ""),
+                "@since {0}".format(since_version),
```

Comment:
This was manually tested as providing the same functionality, the latter patterns would have gotten unruly so I applied the same pattern just for consistency.